src/vaadin-accordion.html

```diff
@@ -34,10 +49,9 @@
        *
        * @memberof Vaadin
        * @mixes Vaadin.ElementMixin
-       * @mixes Vaadin.ThemableMixin
        * @demo demo/index.html
        */
-      class AccordionElement extends Vaadin.ElementMixin(Vaadin.ThemableMixin(Polymer.Element)) {
```

Comment:
what was the reason to remove `Vaadin.ThemableMixin`?

Reply:
Accordion itself has only `<slot>` and does not expose any stylable parts at this point.
So, the `ThemableMixin` is only needed by details (and accordion panel extending it).